frontend/src/Context/PublisherProvider/usePublisher/usePublisher.spec.tsx

```diff
@@ -83,7 +83,7 @@ describe('usePublisher', () => {
 
       const { result } = renderHook(() => usePublisher());
       act(() => {
-        result.current.initializeLocalPublisher();
+        result.current.initializeLocalPublisher({});
```

Comment:
Note: all of these were needed since I made `initializeLocalPublisher`'s parameter of `options` to be required instead of optional